core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -494,15 +488,7 @@ impl SchedulerController {
         bank: &Bank,
     ) -> (u64, u64) {
         let cost = CostModel::calculate_cost(transaction, &bank.feature_set).sum();
-        let fee = bank.fee_structure.calculate_fee(
-            transaction.message(),
-            5_000, // this just needs to be non-zero
-            fee_budget_limits,
-            bank.feature_set
-                .is_active(&include_loaded_accounts_data_size_in_fee_calculation::id()),
-            bank.feature_set
-                .is_active(&remove_rounding_in_fee_calculation::id()),
-        );
+        let reward = bank.calculate_reward_for_transaction(transaction, fee_budget_limits);
```

Comment:
Simply sharing bank's reward calculation

Reply:
there will be some oddity here when the feature gets activated. txs received before feature-activation will still have old calculation (prio 50% burned) those after will not.
This will only be an effect around the epoch boundary.